web/themes/custom/westy/templates/content/node--alert--featured.html.twig

```diff
@@ -0,0 +1,39 @@
+{# Set Alert Type #}
+{% if node.field_severity.value == 20 %}
+    {% set alertType = 'warning' %}
+    {% set TypeName = 'warning' %}
+{% else %}
+    {% set alertType = 'info' %}
+    {% set TypeName = 'information' %}
+{% endif %}
+
+{% set alertIcon %}
+{% include "@westy/icon/icon.twig" with {
+    name: TypeName,
+    size: 'lg',
+    } only %}
+{% endset %}
+
+{# CLASSES #}
+{% set classes= [
+    'bg-' ~ alertType,
+    'text-light',
+]%}
+
+{# TWIG #}
+<div class="{{ classes|join(' ')|trim }}" data-nid="{{ node.id }}" {% if node.changed %} data-changed="{{ node.changed.value }}" {% endif %}>
+    <div class="container">
+        <div class="d-flex flex-row p-3">
+            {{ alertIcon }}
+            <div>
+                <div class="fw-bold">{{ label }}</div>
+                <div class="global-alert-content">{{ content }}</div>
+                <span class="small">
+                    Last updated:
+                    {{ node.changed.value|date('g:i a F j, Y') }}
```

Comment:
Any particular reason for this format? We usually start with Month Day, Year and end with time on every other date format we have in use.

Reply:
Its mirroring what I see on Portland.gov's Alert

Reply:
Ah, that makes sense. And... we should update the Portland.gov alert to use a consistent date format. Please go ahead and match it to the format we have for created and updated on pages. I'll write up a story to address Portland.gov.